hub/api/dataset.py

```diff
@@ -220,19 +220,29 @@ def __iter__(self):
             yield self[i]
 
     def _load_meta(self):
-        if dataset_exists(self.storage):
```

Comment:
Why was this helper removed?

Reply:
i found it kind of confusing, but i can re-add